### PR TITLE
Meson fixes

### PR DESCRIPTION
**Pull request by t-8ch**
Descriptions are in the commit messages.

### CODE REVIEW

File: libblkid/meson.build [hunk 1]
```diff
@@ -141,7 +141,10 @@ lib_blkid = both_libraries(
...
-pkgconfig.generate(lib_blkid,
-                   description : 'Block device id library',
-                   subdirs : 'blkid',
-                   version : pc_version)
+if build_libblkid
+  pkgconfig.generate(lib_blkid,
+                     description : 'Block device id library',
+                     subdirs : 'blkid',
+                     version : pc_version)
+  install_headers(join_paths(meson.current_build_dir(), 'blkid.h'), subdir : 'blkid')
```
Comment (eli-schwartz) [inline libblkid/meson.build:149]: In the above configure_file which creates this, you should use `install: build_libblkid, install_dir: join_paths(get_option('includedir'), 'blkid')`.

File: meson.build [hunk 2]
```diff
@@ -26,6 +26,7 @@ usrbin_exec_dir = join_paths(execprefixdir, 'bin')
 usrsbin_exec_dir = join_paths(execprefixdir, 'sbin')
 sysconfdir = '/etc'
 localstatedir = '/var'
+bashcompletiondir = join_paths(prefixdir, get_option('datadir'), 'bash-completion', 'completions')
```
Comment (eli-schwartz) [inline meson.build:29]: bash-completion has a pkg-config file, e.g. `pkg-config bash-completion --variable=completionsdir`

**Comment by karelzak** [comment]
Thank you for working on the meson!

**Comment by karelzak** [comment]
It works locally, but built by GitHub Actions does not like it (see https://github.com/util-linux/util-linux/runs/4670142544?check_suite_focus=true), it ends with:
```
Program /home/runner/work/util-linux/util-linux/tools/meson-make-symlink.sh found: YES (/home/runner/work/util-linux/util-linux/tools/meson-make-symlink.sh)

meson.build:981:10: ERROR: Function does not take positional arguments.

A full log can be found at /home/runner/work/util-linux/util-linux/build/meson-logs/meson-log.txt
Error: Process completed with exit code 1.
```

**Comment by karelzak** [comment]
BTW, would be better to introduce bashcompletions_links = {} (like we have manlinks = {}) and call meson.add_install_script() later in some loop for all the bashcompletions_links ?

**Pull request merged**